contracts/ronin/slash-indicator/SlashUnavailability.sol

```diff
@@ -59,8 +65,19 @@ abstract contract SlashUnavailability is ISlashUnavailability, HasValidatorContr
         _slashAmountForUnavailabilityTier2Threshold
       );
     } else if (_count == _unavailabilityTier1Threshold) {
-      emit Slashed(_validatorAddr, SlashType.UNAVAILABILITY_TIER_1, _period);
-      _validatorContract.execSlash(_validatorAddr, 0, 0);
+      bool _tier1SecondTime = checkBailedOutAtPeriod(_validatorAddr, _period);
+      if (!_tier1SecondTime) {
+        emit Slashed(_validatorAddr, SlashType.UNAVAILABILITY_TIER_1, _period);
+        _validatorContract.execSlash(_validatorAddr, 0, 0);
+      } else {
+        /// Handles tier-3
+        emit Slashed(_validatorAddr, SlashType.UNAVAILABILITY_TIER_3, _period);
+        _validatorContract.execSlash(
+          _validatorAddr,
+          block.number + _jailDurationForUnavailabilityTier2Threshold,
+          _slashAmountForUnavailabilityTier2Threshold
+        );
+      }
```

Comment:
This makes sense only if the indicator resets on bailing out:
https://github.com/axieinfinity/ronin-dpos-contracts/blob/1d3f5e3c1de471edd6e8b4ea15167130f40e3d90/contracts/ronin/slash-indicator/CreditScore.sol#L90